test/checkpoint/test_checkpoint.c

```diff
@@ -601,8 +673,9 @@ usage(void)
 {
     fprintf(stderr,
       "usage: %s\n"
-      "    [-DmpvXx] [-C wiredtiger-config] [-c checkpoint] [-h home] [-k keys] [-l log]\n"
-      "    [-n ops] [-r runs] [-s 1|2|3|4|5] [-T table-config] [-t f|r|v] [-W workers]\n",
+      "    [-DmpRvXx] [-C wiredtiger-config] [-c checkpoint] [-h home] [-k keys] [-l log]\n"
+      "    [-n|N ops|stable-timestamp] [-r runs] [-s 1|2|3|4|5] [-T table-config] [-t f|r|v]\n"
```

Comment:
The `[-n|N ops|stable-timestamp]` doesn't look right, I don't see a `N` option supported.

Reply:
Ahh, I was initially using -N <stable-timestamp> to provide stop-ts, then I matched with your changes in schema-abort to use -S <stop_ts>, but forgot to make this change. Good catch. I have fixed this.